eclipse-projects/lib-test/.classpath

```diff
@@ -1,92 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-        <!-- This file is updated by the UpdateEclipseClasspath app. -->
```

Comment:
This changes are in PR #213, which questions if we even want these .classpath files in version control since they are autogenerated. We may want to remove these.